src/main/java/frc/robot/AutoCommands.java

```diff
@@ -171,8 +163,8 @@ public Command babyBird() {
 
   public Command shootMid() {
     return new ShootTeleopCommand(
-          drivetrainSubsystem, shooterSubsystem, turretSubsystem, ledSubsystem, desiredX,
-          desiredY, STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)
+          drivetrainSubsystem, shooterSubsystem, turretSubsystem, ledSubsystem, controlBindings.translationX(),
+          controlBindings.translationY(), STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)
```

Comment:
Don't create a new instance of ControlBindings, just take the suppliers as parameters to the method.

```java
  /**
   * Comand to shoot a note into the middle of the field
   * @param xSupplier X translation supplier
   * @param ySupplier Y translation supplier
   * @return new command
   */
  public Command shootMid(
      Supplier<Measure<Velocity<Distance>>> xSupplier, Supplier<Measure<Velocity<Distance>>> ySupplier) {
    return new ShootTeleopCommand(
        drivetrainSubsystem, shooterSubsystem, turretSubsystem, ledSubsystem, xSupplier,
        ySupplier, STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)
            .deadlineWith((new LEDAlternateCommand(ledSubsystem, Color.kBlue, Color.kOrange, Seconds.of(0.1))));
  }
```
Then you can use the existing controlBindings in RobotContainer:
```java
    controlBindings.babyBomber().ifPresent(trigger -> trigger.whileTrue(
        autoCommands.babyBird()
            .andThen(autoCommands.shootMid(controlBindings.translationX(), controlBindings.translationY()))
            .repeatedly()));
```